analytics/index.yaml

```diff
@@ -652,6 +652,18 @@ paths:
           minimium: -180
           maximum: 180
           example: -74.0023705
+        - name: uia
+          in: query
+          description: |
+            Allows to track user anonymously.
+
+            When set to 1 all IP bytes will be masked (0.0.0.0), no GeoIP data will be stored
+            and visits won't be matched by the fingerprinting process. When set back to 0 (or ommited)
+            all available visitor data will be enriched.
```

Comment:
When values were enumerated I was trying to describe them in a list. Check how `send_image` param is described in this file for example.

OpenAPI has also `default` attribute for params.